guides/wide-ep-lws/manifests/modelserver/base/prefill.yaml

```diff
@@ -119,24 +119,23 @@ spec:
           startupProbe:
             httpGet:
               path: /health
-              port: metrics
-            initialDelaySeconds: 0
+              port: 8000
             periodSeconds: 1
-            timeoutSeconds: 5
-            failureThreshold: 1800
+            timeoutSeconds: 3
+            failureThreshold: 360
```

Comment:
This could be in excess of 45 minutes (if not longer).

Please update wide-ep startup probes to take `45 * 60` failureThreshold.